cgmanifest.json

```diff
@@ -3874,249 +4314,409 @@
         }
       }
     },
     {
       "component": {
         "type": "other",
         "other": {
-          "name": "nmap",
-          "version": "7.90",
-          "downloadUrl": "https://nmap.org/dist/nmap-7.90.tar.bz2"
+          "name": "oath-toolkit",
+          "version": "2.6.2",
+          "downloadUrl": "https://download.savannah.gnu.org/releases/oath-toolkit/oath-toolkit-2.6.2.tar.gz"
         }
       }
     },
     {
       "component": {
         "type": "other",
         "other": {
-          "name": "node-problem-detector",
-          "version": "0.8.8",
-          "downloadUrl": "https://github.com/kubernetes/node-problem-detector/archive/refs/tags/v0.8.8.tar.gz"
+          "name": "omi",
+          "version": "1.6.6",
+          "downloadUrl": "https://github.com/microsoft/omi/archive/v1.6.6-0.tar.gz"
         }
       }
     },
     {
       "component": {
         "type": "other",
         "other": {
-          "name": "nodejs",
-          "version": "14.17.0",
-          "downloadUrl": "https://nodejs.org/download/release/v14.17.0/node-v14.17.0.tar.xz"
+          "name": "oniguruma",
+          "version": "6.9.5",
+          "downloadUrl": "https://github.com/kkos/oniguruma/releases/download/v6.9.5/onig-6.9.5.tar.gz"
         }
       }
     },
     {
       "component": {
         "type": "other",
         "other": {
-          "name": "npth",
-          "version": "1.6",
-          "downloadUrl": "https://github.com/gpg/npth/archive/npth-1.6.tar.gz"
+          "name": "open-vm-tools",
+          "version": "11.1.0",
+          "downloadUrl": "https://github.com/vmware/open-vm-tools/releases/download/stable-11.1.0/open-vm-tools-11.1.0-16036546.tar.gz"
         }
       }
     },
     {
       "component": {
         "type": "other",
         "other": {
-          "name": "nspr",
-          "version": "4.21",
-          "downloadUrl": "https://archive.mozilla.org/pub/nspr/releases/v4.21/src/nspr-4.21.tar.gz"
+          "name": "OpenIPMI",
+          "version": "2.0.25",
+          "downloadUrl": "https://sourceforge.net/projects/openipmi/files/latest/download/openipmi-2.0.25.tar.gz"
         }
       }
     },
     {
       "component": {
         "type": "other",
         "other": {
-          "name": "nss",
-          "version": "3.44",
-          "downloadUrl": "https://archive.mozilla.org/pub/security/nss/releases/NSS_3_44_RTM/src/nss-3.44.tar.gz"
+          "name": "openjdk8",
+          "version": "1.8.0.292",
+          "downloadUrl": "https://github.com/AdoptOpenJDK/openjdk-jdk8u/archive/jdk8u292-b10.tar.gz"
         }
       }
     },
     {
       "component": {
         "type": "other",
         "other": {
-          "name": "nss-altfiles",
-          "version": "2.23.0",
-          "downloadUrl": "https://github.com/aperezdc/nss-altfiles/archive/v2.23.0.tar.gz"
+          "name": "openjdk8",
+          "version": "1.8.0.212",
```

Comment:
I think this one should go before the version above.

Reply:
correct - fixed